<launch>
    <arg name="ars548_demo" default="123"/>

    <node pkg="ars548_process" type="ars548_process_node" name="" output="screen">
    </node>
    <!-- <node pkg="ars548_process" type="test_radar_input_node" name="">
    </node> -->
    <node pkg="ars548_process" type="info_convert_node" name="">
    </node>
    <!--node pkg="rviz" type="rviz" name="rviz" output="screen" args="-d $(find ars548_process)/launch/ars548_display.rviz">
    </node-->

</launch>